src/barco.c

```diff
@@ -44,8 +44,8 @@ int main(int argc, char **argv) {
                      "directory to mount as root in the container"),
       cmd =
           arg_strn("c", "cmd", "<s>", 1, 1, "command to run in the container"),
-      arg =
-          arg_strn("a", "arg", "<s>", 0, 1, "argument to pass to the command"),
+      arg = arg_strn("a", "arg", "<s>", 0, 2097152, // check it by `getconf _POSIX_ARG_MAX`
```

Comment:
Is it possible to use `_POSIX_ARG_MAX` directly? That way we don't need the comment.

Reply:
Good point, I defined `_POSIX_ARG_MAX`